hackerrank/prob6/file.js

```diff
@@ -0,0 +1,43 @@
+// make some javascript objects
+
+
```

Comment:
you could make the outputs in the problem look the same as what they are calling in the hackerrank answer by setting area to this.area in the constructor rather than making a function.